src/realm/object-store/audit.mm

```diff
@@ -1038,14 +1043,15 @@ void validate_metadata(std::vector<std::pair<std::string, std::string>>& metadat
     for (size_t i = 0, count = tv.size(); i < count; ++i)
         objects.push_back(tv.get_key(i));
 
-    m_current_scope->events.push_back(
-        audit_event::Query{now(), version, tv.get_target_table()->get_key(), std::move(objects)});
+    for (auto& scope : m_current_scopes) {
+        scope->events.push_back(audit_event::Query{now(), version, tv.get_target_table()->get_key(), objects});
```

Comment:
Not sure how this ends up being used, but it might be best to only create one instance outside the loop so that the identical values are pushed to each scope. Just in case users want to compare events by value and the time is off by a nanosecond. (here and elsewhere)

Reply:
I don't think it'll matter for how the current customer is using it, but that seems like a reasonable thing to guarantee.